### DISCUSSION TITLE
GitHub pages does not work

### DISCUSSION THREAD

**Discussion by Oleh-Kliapko** [community/community#101826] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hello all.
Today my links of all my sites which are built on GitHub pages do not work!
What is happened? Help me please

For instance:
https://oleh-kliapko.github.io/GooseTrack_front/
https://oleh-kliapko.github.io/goit-markup-hw-08/
and others

**Comment by AidenJamesYt** [comment, marked as answer]
Have you published it?

**Reply by Oleh-Kliapko** [reply]
Seems I understood. I changed names of repos, but there are old names in package.json. Now I am trying to fix it because the sites already load but with error: the server responded with a status 404

**Reply by Lanrad23** [reply]
In addition to what is posted above, if you're using a custom domain, verify that your DNS settings are correctly configured. A misconfiguration or expired domain can lead to your site being inaccessible. Check if the domain is still pointing to the GitHub Pages servers. Besides, sometimes, your browser or DNS cache might cause issues. Try accessing your sites from a different browser, incognito mode, or after clearing your browser's cache.

**Comment by DarrenChochian** [comment]


**Reply by Oleh-Kliapko** [reply]
Thanks for advice. I guessed and have already done it

**Comment by tsotneforester** [comment]
same for my repos. all live pages are broken. have not done anything like changing repo name. they all stopped working!
